tests/backend/test_backend_test.py

```diff
@@ -6,6 +6,7 @@
 import os
 import platform
 import unittest
+from pathlib import Path
```

Comment:
```suggestion
import pathlib
```

Import modules only: https://google.github.io/styleguide/pyguide.html#224-decision